cli/src/declarations.ts

```diff
@@ -297,6 +297,17 @@ export interface CapacitorConfig {
      * @default true
      */
     resolveServiceWorkerRequests?: boolean;
+
+    /**
+     * If set to "force", margins will be adjusted for edge to edge regardless of any other settings.
+     * If set to "auto", or is missing, will check for Android 15 and the setting of [windowOptOutEdgeToEdgeEnforcement](https://developer.android.com/reference/android/R.attr#windowOptOutEdgeToEdgeEnforcement) and decide based on those.
```

Comment:
would be good to explain the actual behavior instead of just saying "decide based on those"